preprocessing/tabular_data/training_model_on_tabular_data.ipynb

```diff
@@ -0,0 +1,299 @@
+{
```

Comment:
<p>extra space before "now".</p><p> </p><p>Should be "an S3 location"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1466/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>